### ISSUE TITLE
Family of Docs content

### ISSUE THREAD

**Issue by nicholascar** [w3c/dxwg#364]
This issue was created in the [Profile Guidance document](https://w3c.github.io/dxwg/profiles/) and is listed in it. Once consensus on addressing it is reached here in comments below, the results will be added to the document and the issue closed.

(Antoine:) I've kept Nick's listing but I have doubts that DCAT fits here. I see more value in listing only the 3 profile-related docs, as they are meant to work together.

**Comment by kcoyle** [comment]
This is about the use of "Family of documents" section that includes DCAT and the three profile documents (profgui, profileDesc, conneg). See also:

[Karen's comment](https://lists.w3.org/Archives/Public/public-dxwg-wg/2018Sep/0255.html) 

[Simon's response](https://lists.w3.org/Archives/Public/public-dxwg-wg/2018Sep/0260.html) suggesting using "DXWG family of documents"

(Me: does the concept of DXWG fade after the group ends?)

**Comment by dr-shorthair** [comment]
@kcoyle legitimate issue. Though I think DXWG is a better title within document drafts, for the time-being.

**Comment by aisaac** [comment]
For the time being @dr-shorthair 's suggestion is good.
This said my very issue is whether there should be a same 'family of documents' section in the DCAT spec and in the profiles specs. This is nice for selling our work but it may not be very helpful for readers. Well, maybe it can help in DCAT, I don't know. But for the profile specs, I currently don't see why we should hint that readers should be also interested in reading the DCAT one (other than for their general knowledge).

**Comment by aisaac** [comment]
Resolved at F2F 26 Oct: only the group of profile documents should appear in each of the profile-related documents; the DCAT document will remove the family of documents section
https://www.w3.org/2018/10/26-dxwg-minutes.html#x05

**Comment by aisaac** [comment]
I've tried to implement this resolution at https://github.com/w3c/dxwg/pull/506.
The result is visible at https://raw.githack.com/w3c/dxwg/guidance-new-document-overview/profiles/index.html#ProfileDocs
Hopefully it works for all documents.
Two notes:
- I've refrained from indicating the formal status of the documents now. It makes the section more robust wrt what will happen in the coming months.
- I've considered moving the overview to inside the intro, where it would consume less space. For the moment I've not done it, but feedback on this point is welcome!
